packages/macro/test/jsx-trans.ts

```diff
@@ -216,6 +216,20 @@ export default [
         <Trans id="msg.hello" />;
       `,
   },
+  {
+    name: "Production - import type doesn't interference on normal import",
+    production: true,
+    useTypescriptPreset: true,
+    input: `
+        import { withI18nProps } from '@lingui/react'
```

Comment:
You are not using `import type` in your tests

Reply:
It wasn't related to `import type`. The same problem appeared even with regular import. I've checked it locally that with new changes it works (with regular and type import).